README.md

```diff
@@ -54,8 +54,22 @@ https://www.digitalocean.com/community/articles/geddy-js-a-no-brainer-mvc-node-j
 
 #### License
 
-Apache License, Version 2
+Copyright 2017. Geddy - Web-app development framework
```

Comment:
@ckhatton , seeing this comment of @mde, possibly this is one explanation why this project did not get much traction. The problem is that he even does not realize that he sounds nuts (see #731).

So, if you want help to cleanup this **SHIT**, add me, and I do it. You should really respect the time of the potential users more.